src/shared/services/map-location/index.js

```diff
@@ -1,28 +1,114 @@
-function mapLocation(loc) {
+export const location2feature = location => ({
+  type: 'Point',
+  coordinates: [location.lng, location.lat],
+});
+
+export const feature2location = feature => {
+  const { coordinates } = feature;
+  return {
+    lat: coordinates[1],
+    lng: coordinates[0],
+  };
+};
+
+/**
+ * converts the sia location in latlon fromat
+ */
+export function mapLocation(loc) {
   const location = {};
 
-  if (loc.dichtstbijzijnd_adres) {
-    location.address = { ...loc.dichtstbijzijnd_adres };
-    location.address.huisnummer = `${location.address.huisnummer}`;
-    location.address.huisnummer_toevoeging = `${location.address.huisnummer_toevoeging}`;
+  if (loc.geometrie) {
+    location.location = feature2location(loc.geometrie);
+  }
+
+  if (loc.buurt_code) {
+    location.buurtcode = loc.buurt_code;
   }
 
-  if (loc.omgevingsinfo) {
-    location.buurt_code = loc.omgevingsinfo.buurtcode;
-    location.stadsdeel = loc.omgevingsinfo.stadsdeelcode;
+  if (loc.stadsdeelcode) {
+    location.stadsdeelcode = loc.stadsdeel;
   }
 
-  if (loc.query) {
-    location.geometrie = {
-      type: 'Point',
-      coordinates: [
-        loc.query.longitude,
-        loc.query.latitude,
-      ],
-    };
+  if (loc.address) {
+    location.address = address2pdok(loc.address);
   }
 
   return location;
 }
 
+export const pdok2address = pdokAddress => {
```

Comment:
Are we using PDOK services anywhere in the SIA application?

Reply:
not yet, but will do in the next pr :-)

Reply:
Then please add only the functionality that we need in this PR. Makes the coming PRs much harder to review otherwise.